lib/ansible/modules/storage/netapp/na_ontap_quotas.py

```diff
@@ -326,7 +326,7 @@ def apply(self):
                 elif cd_action == 'delete':
                     self.quota_entry_delete()
                 elif modify_quota is not None:
-                    for key in modify_quota:
+                    for key in list(modify_quota):
```

Comment:
@mattclay Matt, can you explain why this is needed?   We're using this pattern in many places, why is it an issue here?

Reply:
It is due to this upstream cpython change: https://github.com/python/cpython/pull/12596

Instead of only tracking if the size of a dictionary changes, cpython 3.8 is also now tracking if any key names change as well while iterating.